ai-service/extract.py

```
@@ -0,0 +1 @@
+import json; from app.main import app; print(json.dumps(app.openapi()))
```

Ruff (0.8.2)

Multiple statements on one line (semicolon)

(E702)